01week/datatypes.js

```diff
@@ -1,3 +1,4 @@
+//  doing a new thing 
```

Comment:
On line 60, there is a parentheses missing that is causing an error. A parentheses around the variable is unnecessary. 
```javascript
if (arg1 || !arg2){ 
   ...
}
```

Reply:
The parentheses around the variables in the last function are unnecessary as well. Please remove